### ISSUE TITLE
Use simppl to access BLE advertisement messages?

### ISSUE THREAD

**Issue by mbeltoft** [martinhaefner/simppl#33]
Is it possible to use simppl to call the StartDiscovery method in Bluez, and then listen for BLE advertisement messages?

**Comment by martinhaefner** [comment]
Yes, this should be possible with simppl. Calling StartDiscovery you just
need a simppl Stub with appropriate object path set. I don't have any
example code for this, just for advertising a BLE GATT service. Do you know
how to wait for BLE advertisement messages on D-BUS? I haven't found
anything in a hurry.

Regards,
Martin

Am Di., 8. Aug. 2023 um 23:26 Uhr schrieb Mogens Beltoft <
***@***.***>:
